SQL/vetclinicData.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN TRANSACTION;
+INSERT INTO "clients" ("client_id","first_name","last_name","phone","email") VALUES (1,'john','doe','555-321-6543','johndoe@mail.com');
+INSERT INTO "pets" ("pet_id","name","species","gender","dob","owner") VALUES (1,'Isis','cat','FS','2006-11-01',1);
+INSERT INTO "appointments" ("date","time","client","pet","reason") VALUES ('2020-05-03','2020-05-03 11:00:00',1,1,'Annual Exam');

```

Comment:
When we display the date and time in the appointments table is it going to have the date repeated? If the date has to be on the time, can we just use datetime instead of separating them?

Reply:
Are we also going to display AM/PM? I know as a RN they used military time.

Reply:
If we changed the name of Isis the cat to Lily, line 3 would have to be changed as well.

Reply:
Mia, the comment to change the cat's name is on line 3.

Reply:
^ oof. I knew that. /facepalm

Reply:
The date does have to be on the time. However, I can insert just a TIME (e.g 11:00) and then it gives it some random date. When I was messing around it gave it something in the year 2000. When I pull from that field I can just pull time() and it'll spit out '11:00'. Figured I'd do that to populate time. Not sure if the same thing exists to just pull the YYYY-MM-DD from the same field and not the time. The 24-hour clock is the easiest way to insert and compare times, so I'm going to keep that.